visualization/autoware_perception_rviz_plugin/CMakeLists.txt

```
@@ -4,7 +4,12 @@ project(autoware_perception_rviz_plugin)
 find_package(autoware_cmake REQUIRED)
 autoware_package()
 
+find_package(autoware_perception_msgs REQUIRED)
+find_package(pluginlib REQUIRED)
 find_package(Qt5 REQUIRED COMPONENTS Widgets)
+find_package(rviz_common REQUIRED)
+find_package(rviz_default_plugins REQUIRED)
+find_package(visualization_msgs REQUIRED)
```

You shouldn't need find_package() for ROS packages since autoware_package() in line 5 should be calling ament_auto_find_build_dependencies()

```suggestion
find_package(pluginlib REQUIRED)
find_package(Qt5 REQUIRED COMPONENTS Widgets)
```